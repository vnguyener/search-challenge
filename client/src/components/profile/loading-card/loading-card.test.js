import React from 'react';
import renderer from 'react-test-renderer';

import LoadingCard from '.';

describe('profile loading card component', () => {
  it('renders correctly', () => {
    const element = renderer.create(<LoadingCard />).toJSON();
    expect(element).toMatchSnapshot();
  });
});
