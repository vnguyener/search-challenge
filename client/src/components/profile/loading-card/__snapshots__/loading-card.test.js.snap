// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`profile loading card component renders correctly 1`] = `
<div
  className="loading-card-container"
>
  <div
    className="MuiPaper-root MuiCard-root loading-card MuiPaper-elevation1 MuiPaper-rounded"
  >
    <span
      className="MuiSkeleton-root MuiSkeleton-rect loading-card-media MuiSkeleton-wave"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <div
      className="MuiCardContent-root"
    >
      <span
        className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave"
        style={
          Object {
            "height": 10,
            "marginBottom": 6,
            "width": undefined,
          }
        }
      />
      <span
        className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave"
        style={
          Object {
            "height": 10,
            "width": "80%",
          }
        }
      />
    </div>
  </div>
</div>
`;
